tools/cli/domainCommands.go

```diff
@@ -854,10 +922,12 @@ func (d *domainCLIImpl) migrationDynamicConfigCheck(c *cli.Context) DomainMigrat
 	}
 
 	for _, configKey := range resp {
+		var mismatchedConfig MismatchedDynamicConfig
+
 		if len(configKey.Filters()) == 1 && configKey.Filters()[0] == dc.DomainName {
```

Comment:
add comment on this case, for example: // validate dynamic configs with only domainName filter